core/src/test/java/org/testcontainers/containers/FixedHostPortGenericContainerTest.java

```diff
@@ -0,0 +1,19 @@
+package org.testcontainers.containers;
+
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class FixedHostPortGenericContainerTest {
```

Comment:
hm sorry, I think we had a misunderstanding. 
This is an integration test since it starts a container. Starting it is not necessary, unit test is enough.

If you give me [write permission to the PR](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests), I can do it for you.

Reply:
It configures it, but it doesn't start. -- at least that was my intention

Reply:
Anyway, i am open to suggestions. or examples i should follow

Reply:
@kiview i am open to your suggestions